airbyte-scheduler/persistence/src/main/java/io/airbyte/scheduler/persistence/DatabaseSchema.java

```diff
@@ -50,7 +50,7 @@
   // AirbyteMetadata
   AIRBYTE_METADATA("AirbyteMetadata.yaml");
 
-  static final Path KNOWN_SCHEMAS_ROOT = JsonSchemas.prepareSchemas("tables", DatabaseSchema.class);
+  static final Path KNOWN_SCHEMAS_ROOT = JsonSchemas.prepareSchemas("job_tables", DatabaseSchema.class);
```

Comment:
looks like some of the schemas from `config_tables` were missing before?

Reply:
This file is for the Jobs Database only actually. Previously only the Jobs data are in the database. If I add the config table schema here, some tests will be broken.

I will rename it and create one for the Configs Database in my migration PR.